README.md

```diff
@@ -1,7 +1,7 @@
-# srv-rs
+# srv-rs-upd
```

Comment:
What does `upd` mean? Can we come up with a better name (and also rename this repo to match)?